### ISSUE TITLE
Create an issue hidden from the board

### ISSUE THREAD

**Issue by naholyr** [huboard/huboard#89]
Hello,

As a new user I thought that the issues I did **not** tagged as `0-Backlog` would **not** appear in my backlog. I discovered that **all** issues with no _huboard-tag_ will be listed in backlog.

I imagine how this can be wishable and useful, but in my case (and I think other cases) I use GitHub issues for many things at once:
- Sprint tasks (Huboard rocks at this)
- TRAC-like issues
- TODO-like issues (like long term tasks "as soon as possible" that will not be planned in a sprint)

Maybe you could add an option in huboard settings to choose if we want to put all issues in backlog, or really only the ones tagged with index 0.

##

**Comment by discorick** [comment]
@rauhryan can this be closed?

**Comment by dbolkensteyn** [comment]
I don't think this can be closed : I've just tried to add a `@huboard: :show_all: => false` label to my repo, and I still see all unlabelled issues under `Backlog` in Huboard

**Comment by discorick** [comment]
@rauhryan, could you you take a look at this ?
